docs/boards/Board - MATEKF722-SE.md

```diff
@@ -0,0 +1,91 @@
+# Board - Matek F405-CTR
```

Comment:
This probably needs updating. :-)

Reply:
Thanks. I am learning how to make a PR in Git :-)